File: backend/src/main/java/cohappy/backend/model/dto/response/ChatMessageDTO.java
package cohappy.backend.model.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
public class ChatMessageDTO {
    public String message;
    public byte[] messageImmage;
    private String userCode;
    private byte[] userImage;
    public LocalDateTime timestamp;
}

File: frontend/app/src/main/java/cohappy/frontend/model/dto/response/ChatMessageDTO.kt
package cohappy.frontend.model.dto.response

import java.time.LocalDateTime
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class ChatMessageDTO(
    var message: String? = null,
    var messageImmage: ByteArray? = null,
    var userCode: String? = null,
    var userImage: ByteArray? = null,
    var timestamp: LocalDateTime? = null
)